build/yamls/base/crds-rbac.yml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: aggregate-security-crds-edit
```

Comment:
Is it a convention to name a role as " aggregate-xxx-edit"?

Change "security-crds" to "antrea-policy"?

Reply:
followed example from doc https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles

updated with aggregate-antrea-policies-edit

Reply:
From the examples in the doc, it might be something like antrea-policy-editor?

Reply:
i looked at the example used to aggregate a CRD to edit and view roles.. specifically this one

```
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: aggregate-cron-tabs-edit
  labels:
    # Add these permissions to the "admin" and "edit" default roles.
    rbac.authorization.k8s.io/aggregate-to-admin: "true"
    rbac.authorization.k8s.io/aggregate-to-edit: "true"
rules:
- apiGroups: ["stable.example.com"]
  resources: ["crontabs"]
  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
---
kind: ClusterRole
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  name: aggregate-cron-tabs-view
  labels:
    # Add these permissions to the "view" default role.
    rbac.authorization.k8s.io/aggregate-to-view: "true"
rules:
- apiGroups: ["stable.example.com"]
  resources: ["crontabs"]
  verbs: ["get", "list", "watch"]
```

Reply:
Ok. Looks good to me then.